microsite/data/plugins/rollbar.yaml

```diff
@@ -0,0 +1,8 @@
+---
+title: Rollbar
+author: Andrew Thauer
```

Comment:
Maybe it would make sense to have this as a link to the user/org github page. For example `@andrewthauer` would map to https://github.com/andrewthauer.

Reply:
I believe listing the company that authored the plugin is going to be the default. In order to both support companies and individuals I suggest we make this field a bit more granular:

```
author: @andrewthauer
authorURL: https://github.com/andrewthauer
```
or 

```
author: Spotify
authorURL: https://www.spotify.com
```

Reply:
Made the author a link to authorUrl - https://www.loom.com/share/ea6b4dfef2e04015be0889aba7307898.